src/conf.py

```diff
@@ -37,11 +37,6 @@
 exclude_patterns = []
 
 rst_prolog = """
-.. caution::
-
-   You're reading a draft of the Ferrocene Language Specification. Some parts
-   of this document might be missing, incomplete or incorrect. Our aim is to
-   have the specification ready by the end of 2022.
 """
```

Comment:
You should remove the whole prolog variable.

Reply:
That was my initial reaction. But then I though, what if something else is using `rst_prolog`. I grepped within the repo and did not find anything that uses it, yet the variable is still needed. I found this a bit suspicious, so I decided to leave the variable in.

Let us see what the CI will say...

Reply:
`rst_prolog` is not present in the rest of the repository because it's [a Sphinx configuration variable](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-rst_prolog) used by Sphinx itself. Since it contains no content it's safe to remove it.